CHANGES.rst

```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-19)
+--------------------------
+
+- Moves job controller to the same Kubernetes pod with the
+  REANA-Workflow-Engine-* (sidecar pattern).
+- Adds support for HTCondor and Slurm compute backends.
+- Adds sidecar container to the Kubernetes job pod if Kerberos authentication
+  is required.
+- Provides user secretes mechanism to the job container.
```

Comment:
s/secretes/secrets

We could say perhaps better: "Provides user secrets to the job container runtime tasks" or somesuch